src/ui/shared/BrutalistUI.jsx

```
@@ -0,0 +1,929 @@
+// 5. Brutal Amp Fader (Custom Slider)
+export const BrutalFader = ({ label, initialValue = 7, max = 10 }) => {
+  const [val, setVal] = useState(initialValue);
+  const segments = Array.from({ length: max }, (_, i) => i + 1);
+
+  return (
+    <div className="w-full max-w-sm flex flex-col gap-2">
+       <div className="flex justify-between items-end">
+        <span className="text-xs tracking-widest uppercase opacity-80">{label}</span>
+        <span className="text-sm font-bold text-[#39FF14]">{val}</span>
+      </div>
+      <div className="flex gap-1 h-8 items-end cursor-pointer group" onMouseLeave={() => {}}>
```

Biome (2.4.4)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)